src/terminal/osc.zig

```diff
@@ -788,6 +796,21 @@ pub const Parser = struct {
                 else => self.showDesktopNotification(),
             },
 
+            .conemu_sleep => switch (c) {
+                ';' => {
+                    self.command = .{ .sleep = 100 };
+                    self.buf_start = self.buf_idx;
+                    self.complete = true;
+                    self.state = .conemu_sleep_value;
+                },
+                else => self.state = .invalid,
+            },
+
+            .conemu_sleep_value => switch (c) {
+                '0'...'9' => {},
```

Comment:
I could construct the number directly here, but I thought it would be safer to handle potential overflow correctly by using `std.fmt.parseUnsigned`.

Reply:
`parseUnsigned` allows `_` as in `1_000_000` is a valid Zig number so I always feel a little icky when using it to parse data. I think that doing it like this is fine. Performance shouldn't be a concern.